qa/runtests.sh

```diff
@@ -44,7 +43,7 @@ check_python_version
 
 $(which pip3) install pytest-cov
 
-PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=$(pwd) python3 "$(which pytest)" --cov=. --cov-config .coveragerc --cov-report term-missing --cov-fail-under=$COVERAGE_THRESHOLD -vv tests/
+PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=$(pwd) python3 "$(which pytest)" --cov=./src --cov-config .coveragerc --cov-report term-missing --cov-fail-under=$COVERAGE_THRESHOLD -vv tests/
```

Comment:
there is a mechanism to exclude tests folder and other unwanted folders from test coverage

Reply:
was discussing this with @samuzzal-choudhury . We couldnt get any better solution. Other repos too use this